src/documentation/pages/Dialog/examples/DialogExampleDefaultJS.hbs

```diff
@@ -0,0 +1,52 @@
+<script type="text/javascript">
+(function(){
+  var example = document.querySelector(".docs-DialogExample-default");
+  var button = example.querySelector(".docs-DialogExample-button");
+  var dialog = example.querySelector(".ms-Dialog");
+  var label = example.querySelector(".docs-DialogExample-label")
+  
+  var checkBoxElements = example.querySelectorAll(".ms-CheckBox");
+  var actionButtonElements = example.querySelectorAll(".ms-Dialog-action");
+  var checkBoxComponents = [];
+  var actionButtonComponents = [];
+  
+  // Wire up the dialog
+  var dialogComponent = new fabric['Dialog'](dialog);
+  
+  // Wire up the checkBoxes
+  for(var i = 0; i < checkBoxElements.length; i++) {
+      checkBoxComponents[i] = new fabric['CheckBox'](checkBoxElements[i]);
+  }
+  
+  // Wire up the buttons
+  for(var i = 0; i < actionButtonElements.length; i++) {
+      actionButtonComponents[i] = new fabric['Button'](actionButtonElements[i], actionHandler);
+  }
+
+  // When clicking the button, open the dialog
+  button.onclick = function() {
+      openDialog(dialog);
+    };
+    
+  function actionHandler(event) {
+   var labelText = "";
+   var counter = 0;
+   for(var i = 0; i < checkBoxComponents.length; i++) {
+      if(checkBoxComponents[i].getValue()) {
+          counter++;
+      }
+    }
+    labelText += counter + " option(s) selected. " + this.innerText.trim() + " clicked" ;
+    label.innerText = labelText;
+  }
+  
+  function openDialog(dialog) {
+    // Open the dialog
+    dialogComponent.open();
+
+    // Center the dialog
+    dialog.style.top = (window.innerHeight - dialog.offsetHeight) / 2 + 'px';
```

Comment:
Instead of setting an exact pixel value for centering the Dialog, let's make this happen in css. You can add this code below in somewhere on the main dialog and you can basically remove the positioning css.

```
top: 50%;
left: 50%;
transform: translate(-50%, -50%);
```